### DISCUSSION TITLE
useStorage in vercel giving "ENOENT: no such file or directory" error

### DISCUSSION THREAD

**Discussion by impons** [nitrojs/nitro#899] [Q&A] [answered]
I deployed a nuxt 3 project in vercel and i'm using the `useStorage` feature with the next configuration:
```
nitro: {
    storage: {
      'db': {
        driver: 'fs',
        base: '.data/db',
      },
    },
  },
```
Locally works fine but when deployed it is giving me the next error (when the `useStorage` is called ): `ENOENT: no such file or directory, open '/vercel/path0/.data/db/xxxx` .

Any ideas why is this happening and how to solve it?

**Comment by cosbgn** [comment, marked as answer]
You can't use `fs` in Vercel, because it's a serverless environment. You will need to switch to redis or something else.
